README.rst

```diff
@@ -139,11 +139,17 @@ Use the AWS CLI to save that configuration to your local system:
 
 You can verify that settings were stored correctly by checking the files it created in your ``~/.aws/`` directory.
 
-AWS access for running **cloudigrade** inside Docker must be enabled via environment variables. Set the following variables in your local environment *before* you start running in Docker containers. Values for these variables can be found in the files in your ``~/.aws/`` directory.
+AWS access for running **cloudigrade** inside a local OpenShift cluster must be enabled via environment variables. Set the following variables in your local environment *before* you start running in OpenShift. Values for these variables can be found in the files in your ``~/.aws/`` directory.
 
 -  ``AWS_ACCESS_KEY_ID``
 -  ``AWS_SECRET_ACCESS_KEY``
 -  ``AWS_DEFAULT_REGION``
+-  ``AWS_SQS_ACCESS_KEY_ID``
+-  ``AWS_SQS_SECRET_ACCESS_KEY``
+-  ``AWS_SQS_REGION``
+-  ``AWS_SQS_QUEUE_NAME_PREFIX``
+
```

Comment:
Yes, I believe that's all we need. The `AWS_SQS_QUEUE_NAME_PREFIX` makes it so that each of us have our own pseudo-name-spaced queues (we each have _several_) since we're all sharing one master AWS account. If someone doesn't set an appropriate value here, that person risks colliding with another developer, and this is where I've asked people to "play nicely" with this limitation. :)